profile.graphqls

```diff
@@ -92,4 +112,8 @@ extend type Mutation {
 extend type Query {
     # query all task list, order by ProfileTask#startTime descending
     getProfileTaskList(serviceId: ID, endpointName: String): [ProfileTask!]!
+    # query all task profiled segment list
+    getProfileTaskSegmentList(taskID: String): [BasicTrace!]!
+    # analyze profiled segment
+    getProfileAnalyze(segmentId: String!, duration: Duration!): ProfileAnalyzation!
```

Comment:
We should not use Duration here. As you can see, this ProfileAnalyzation is targeting a specific time range in milliseconds always.